src/create-glamorous.js

```diff
@@ -37,40 +73,46 @@ function createGlamorous(splitProps) {
      *   props and the return value is used.
      * @return {ReactComponent} the ReactComponent function
      */
-    function glamorousComponentFactory(...styles) {
+    function glamorousComponentFactory(...styles: Array<Style>) {
       /**
        * This is a component which will render the comp (closure)
        * with the glamorous styles (closure). Forwards any valid
        * props to the underlying component.
        */
-      const GlamorousComponent = withTheme(
+      let FinalGlamorousComponent // eslint-disable-line prefer-const
```

Comment:
Is there a reason that this needs to be `let`? It isn't being reassigned so we can make it `const`.